### ISSUE TITLE
Impossible to load the main class me.aurous.Auros

### ISSUE THREAD

**Issue by JacoNIX97** [AurousGroup/Bug-Tracker#11]
Impossible to load the main class me.aurous.Auros when trying to start aurous on Arch Linux

**Comment by JPFrancoia** [comment]
Same issue here, on Manjaro Linux.

**Comment by JPFrancoia** [comment]
@JacoNIX97 , which java environment are you using ?  java-8-openjdk/jre ? I was using this one, and switched to java-8-jre/jre, available on AUR. I tried to launch the program with sudo also, but ended with the same issue as #5

**Comment by JacoNIX97** [comment]
@JPFrancoia I'm using Java 8 JRE, now I'm tryng to load the class in the .jre file. Checked also the MANIFEST file but nothing "interesting"

**Comment by pikpok** [comment]
After some tries I managed to run Aurous on my x86_64 Arch.
Are you on x86_64? Then use multilib (https://wiki.archlinux.org/index.php/Multilib) and install this JRE: https://aur.archlinux.org/packages/bin32-jre/
You will also need to install some other multilib deps, like `lib32-gconf`, `lib32-libgnutls28`, `lib32-libxss`, and probably some others I forgot.
Then run app like that: `sudo /usr/lib32/jvm/default-runtime/bin/java -jar Aurous.jar`

**Comment by JacoNIX97** [comment]
Lol, it's what I was doing, thanks I'll try it and keep you updated

**Comment by JPFrancoia** [comment]
Yes everybody is on x86_64 I guess :) That's what they said in #5, the errors come from the architecture. They also said a 64 bits will be released tomorrow.

**Comment by JPFrancoia** [comment]
Ok, no more error on the 64 bits version. You only need to install libgcrypt11 from AUR. And you also need to ```sudo chmod -R +x *``` in the folder. Then, you must run the jar file with sudo, and with java-8-jre/jre, not openjdk.

However, that's some disturbing points:
 - why do we need the oracle version of Java to run it ?
 - why do we need sudo to run it ?

**Comment by Mikaela** [comment]
> why do we need the oracle version of Java to run it ?

Vanilla Arch Linux, works fine with openjdk.

> why do we need sudo to run it ?

no idea where you picked that sudo command, I didn't need to use sudo anywhere.

**Comment by JPFrancoia** [comment]
I'm on manjaro. WIth openjdk: ```Impossible to load the main class me.aurous.Aurous```
With java-8-jre/jre and without sudo: nothing happens, the CLI doesn't print anything and no gui showing.

However, I'm not the only one with the sudo issue. See #5
